tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -563,6 +378,9 @@ var _ = Describe("Configuration testing for DPA Custom Resource", func() {
 			},
 			WantError: false,
 		}, nil),
+	}
+
+	awsTests := []TableEntry{
```

Comment:
is it possible to use labels instead? for Example, add in the Makefile:
FILTER_PROVIDER := $(shell echo '! aws-provider && ! gcp-provider && ! azure-provider' | sed -r "s/[&]* [!] $(POVIDER)-provider|[!] $(POVIDER)-provider[&]*//"
And then mark:
Entry("AWS Without Region No S3ForcePathStyle with BackupImages false should succeed", **Label("aws-provider")**, InstallCase{